torch/_torch_docs.py

```diff
@@ -108,6 +108,19 @@ def merge_dicts(*dicts):
         returned Tensor. Default: ``torch.contiguous_format``.
 """
     ),
+    {
+        "sparse_factory_device_note": """\
+.. note::
+
+   If the ``device`` argument is specified, the device of the
```

Comment:
The above makes sense. Notice that the note must apply to all sparse tensor cases where the indices may mean a pair of `crow_indices` and `col_indices` tensors, etc. So, I propose to avoid using `:attr:indices` as follows:
```
   If the ``device`` argument is not specified the device of the given
   :attr:`values` and indices tensor(s) must match. If, however, the
   argument is specified the input Tensors will be converted to the
   given device and in turn determine the device of the constructed
   sparse tensor.
```